### PR TITLE
Patternlab/DP-26226 header mixed keyboard a11y

### PR DESCRIPTION
**Pull request by clairesunstudio**
Fixed:
  - project: Patternlab
    component: HeaderMixed, mainNav.js
    description: A11y - Fix keyboard navigation. (#1759)
    issue: DP-26226
    impact: Patch
Removed:
  - project: Patternlab
    component: Header, mainNav.js
    description: Refactor mainNav JS and remove deprecated Header component from Patternlab. (#1759)
    issue: DP-26226
    impact: Minor

UAT: https://mayflower.digital.mass.gov/patternlab/b/patternlab/header-mixed-keyboard-a11y/patterns/03-organisms-by-template-header-mixed/03-organisms-by-template-header-mixed.html

### CODE REVIEW

File: packages/patternlab/styleguide/source/assets/js/modules/mainNav.js [hunk 1]
```diff
@@ -115,109 +59,123 @@ export default (function (window, document, $, undefined) {
...
+            dropdownIndex = dropdownLinksLength - 1;
+          } else {
+            console.log('down')
+            // key down focus on the first item
+            dropdownIndex = 0;
+          }  
+          console.log(dropdownIndex)
+          console.log($dropdownLinks[dropdownIndex])
+          // not sure why this doesn't work?????
+          $dropdownLinks[dropdownIndex].focus();
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/assets/js/modules/mainNav.js]:
```suggestion
          $dropdownLinks[dropdownIndex].focus(); 

```
Unsure why this line doesn't work. The first up/down keypress is not triggering the focus moving to the last/first item.
Screen Shot 2023-04-10 at 3 00 59 PM

**Review by ygannett — CHANGES_REQUESTED** [review]
The key commands are working beautifully, but when trying to access the sub menu drop-downs with mouse in the horizontal nav, the drop-downs don't stay open as leaving the top menu area.  So, unable to click submenus.  In the hamburger menu, you need to open the submenus, so this is not an issue.

**Comment by clairesunstudio** [comment]
Good catch! Fixed!

https://mayflower.digital.mass.gov/patternlab/b/patternlab/header-mixed-keyboard-a11y/patterns/03-organisms-by-template-header-mixed/03-organisms-by-template-header-mixed.html

**Review by ygannett — APPROVED** [review]
LGTM.

**Comment by ygannett** [comment]
@clairesunstudio After I approved the PR, noticed this backstop failure:  https://output.circle-artifacts.com/output/job/2618e62c-b8f2-408e-8ff4-e64649f58f3e/artifacts/0/backstop/reports/html/index.html
Since this sample page is not relevant to us anymore, would you like to remove it?

**Pull request merged**